### ISSUE TITLE
On new projects, there's a git tags error when launching PRs

### ISSUE THREAD

**Issue by zephraph** [intuit/auto#1187]
**Describe the bug**

Here's a build error: https://app.circleci.com/pipelines/github/artsy/next/4/workflows/0310105f-0602-4f77-b43c-143083e420bf/jobs/7/steps

And the message that is given

```
Error: Running command 'git' with args [describe, --tags, --abbrev=0, ] failed
fatal: No names found, cannot describe anything.
    at ChildProcess.child.on (/snapshot/auto/node_modules/@auto-it/core/dist/utils/exec-promise.js:49:24)
    at ChildProcess.emit (events.js:198:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
```

**To Reproduce**

Create a new repo without tags, open a PR that tries to deploy a canary

**Expected behavior**

Either
1. It catches the error and tells the user what actions to take (create a release on the repo) _or_
2. It handles it gracefully and creates whatever tag it needs

**Comment by kevinwolfdev** [comment]
I spent a lot of time yesterday dealing with this too. I have a package where I wanted to release only canaries until I reach a good shape for v1.0.0.

**Comment by hipstersmoothie** [comment]
I'll take a look at this today!

**Comment by kevinwolfdev** [comment]
@hipstersmoothie is it possible to start a package with `0.0.0` (while on development, but still continuously releasing to canary), and then when it is on good shape release to `1.0.0`? I was testing this yesterday (without releasing to canary, just to `latest). And when I wanted to merge the “final” pr and added the `major` tag to it, the version got bumped to `2.0.0`.

**Comment by hipstersmoothie** [comment]
> is it possible to start a package with 0.0.0 

Yup. If `auto` is unable to find a version it will use `0.0.0` as the start.

>  And when I wanted to merge the “final” pr and added the `major` tag to it, the version got bumped to `2.0.0`.

If the version was `1.0.0` and you merged a pr with `major` the result would be `2.0.0`.

When I start publishing a package with `auto` I usually make sure the version to `0.0.0` or something in the `0.x` range.

**Comment by hipstersmoothie** [comment]
BTW Thanks for choosing `auto` to do your releases! any and all feedback is welcome

**Comment by kevinwolfdev** [comment]
Thanks! I will try it. And the weird thing is that it was releasing all PRs on `0.x.0`, but when I marked one to be `major`, it jumped to `2.0.0`. Will create a repo later to try to replicate the behavior.

**Comment by hipstersmoothie** [comment]
If you have the logs for it I can try to deduce what happened

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.30.4`](https://github.com/intuit/auto/releases/tag/v9.30.4) :rocket:
